pkgs/development/web/deno/default.nix

```diff
@@ -68,7 +46,20 @@ rustPlatform.buildRustPackage rec {
   # The deno_ffi package currently needs libtcc.a on linux and macos and will try to compile it at build time
   # To avoid this we point it to our copy (dir)
   # In the future tinycc will be replaced with asm
-  DENO_FFI_LIBTCC = "${libtcc}/lib";
+  libtcc = tinycc.overrideAttrs (oa: {
+    makeFlags = [ "libtcc.a" ];
+    # tests want tcc binary
+    doCheck = false;
+    outputs = [ "out" ];
+    installPhase = ''
+      mkdir -p $out/lib/
+      mv libtcc.a $out/lib/
+    '';
+    # building the whole of tcc on darwin is broken in nixpkgs
+    # but just building libtcc.a works fine so mark this as unbroken
+    meta.broken = false;
+  });
```

Comment:
Lets move that back into the let in.

Reply:
Its easier to override here

Reply:
I think it would need to use finalAttrs for that because the change won't be propagated into TCC_PATH IIRC.

Reply:
sure but again it's easier than writing out the whole override again yourself. Can't work off the `TCC_PATH`

The only sufficient alternative would be to move it back to let & add it to passthru